package.json

```diff
@@ -15,6 +15,7 @@
     "ethers": "^5.0.1",
     "jazzicon": "^1.5.0",
     "next": "^9.4.4",
+    "node-fetch": "^2.6.0",
```

Comment:
Don't think this is necessary as mentioned in my other comment.

Reply:
Great.